src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -4,6 +4,8 @@
     <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;net7.0</TargetFrameworks>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <AssemblyVersion>6.0.0.0</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <CLSCompliant>false</CLSCompliant>
```

Comment:
This should be true, the assembly level attribute was true before.
![image](https://github.com/user-attachments/assets/231c7ff3-670d-488e-a7e6-46d9ad2bee88)

Reply:
Wrong assembly. I just looked at the 6.0.0 nuget package. It's false there:

![image](https://github.com/user-attachments/assets/de9c424b-6761-4e32-8027-867115312029)

Reply:
But this property can be removed, it doesn't do anything. CLSCompliant is set to false in the .il: https://github.com/dotnet/maintenance-packages/blob/9bade88c9d87659bf1a55c756028d4c6537f9b4b/src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il#L43

Reply:
> But this property can be removed, it doesn't do anything.

If I remove the explicit `<CLSCompliant>false</CLSCompliant>` property from the ilproj, then it acquires the default value, which we are setting to `true` in the root Directory.Build.props. This is because there's nothing in the ilproj indicating that we need to either set it to false or not set it at all.

Before, I had that line that set it to false by default: https://github.com/dotnet/maintenance-packages/pull/135#discussion_r1801649267 but per your suggestion I removed it.

Reply:
> Wrong assembly

Sorry, I meant for my original comment to be on the Vectors assembly which was false at the time, my mistake with comment placement.

As for whether or not this stays in the ilproj - as @ViktorHofer mentions it doesn't do anything.  @carlossanlop if you're worried about the property hanging around with the wrong value you could exclude it for ILProj in the targets.

Reply:
@ViktorHofer thanks for showing the value set in the .il.

> you could exclude it for ILProj in the targets.

Ok, I can do that. Note that I also need it to be set to false in Microsoft.IO.Redist. But I assume it's ok to explicitly set the property to `false` there, as we don't have an .il file to worry about.